### DISCUSSION TITLE
Login issue api/data/v1/form/read 404 Not Found

### DISCUSSION THREAD

**Discussion by Haritest** [Sunbird-Ed/Community#378] [Installation] [answered]
Hi Team,

  We run the PostInstallScript job successfully and Forms were created during the job build. But now getting `api/data/v1/form/read 404 Not Found` while loading the home page. PFA

![image](https://user-images.githubusercontent.com/13552553/216285199-bc0ecd46-6004-4567-9ba8-a4a600447311.png)

![image](https://user-images.githubusercontent.com/13552553/216285963-6d77caf1-9eaa-446e-89da-2993e1e869f9.png)

![image](https://user-images.githubusercontent.com/13552553/216286955-53137835-614c-4fd2-81c0-0e58fa04b60b.png)

@santhosh-tg

**Comment by Krishnaj20** [comment, marked as answer]
@gandham-santhosh - can you look into this query and respond?

**Reply by Krishnaj20** [reply]
@gandham-santhosh - Can you look into this query and respond?

**Comment by gandham-santhosh** [comment]
@Haritest is this issue fixed?  If not can you share the for read request http response from the web browser console?

**Reply by Krishnaj20** [reply]
@Haritest - Is this issue fixed?

**Reply by Krishnaj20** [reply]
@Haritest - marking this as answered as we haven't received an update. Pls feel free to open a new thread in case you still have any follow up queriess.

**Reply by Haritest** [reply]
@gandham-santhosh @Krishnaj20 We stopped and not using this infra now, so we cant check the issue again.
